projects/@shared/components/properties-item-new.vue

```diff
@@ -0,0 +1,305 @@
+        <template v-for="(item, index) in term">
+          <span v-if="item" :key="index" class="properties-item__term">
+            {{ item }}
+          </span>
+        </template>
+      </template>
+
+      <span v-else-if="!showTooltip" class="properties-item__term">
+        {{ term }}
+      </span>
+
+    </dt>
+
+    <dd v-if="isImageValue" class="properties-item__value properties-item__value--image">
+      <img
+        class="properties-item__image"
+        :src="image.url"
+        :alt="image.title || ''"
+      />
+      <external-link :href="image.url">
+        <span>{{ image.href }}</span>
+      </external-link>
+    </dd>
+
+    <dd v-else-if="isLinkValue" class="properties-item__value properties-item__value--link">
+      <external-link :href="value">
+        {{ value }}
+      </external-link>
+    </dd>
+
+    <dd v-else-if="isColorValue" class="properties-item__value properties-item__value--color">
+      <span>{{ value }}</span>
+      <span class="properties-item__color" :style="{ backgroundColor: value }"></span>
+    </dd>
+
+    <dd v-else-if="isUrlsValue" class="properties-item__value properties-item__value--urls">
+      <div v-for="(item, index) in value" :key="index">
+        <template v-if="item.url">
+          <external-link :href="item.url">
+            <span>{{ item.url }}</span>
+          </external-link>
+        </template>
+        <template v-if="item.attributes">
+          <span v-for="(attribute, name) in item.attributes" :key="name">
+            {{ name }}: {{ attribute }}
+          </span>
+        </template>
+      </div>
+    </dd>
+
+    <dd v-else class="properties-item__value">
+      <span v-if="value">{{ value }}</span>
+    </dd>
+  </div>
+</template>
+
+<script>
+import AppTooltip from './app-tooltip';
+import ExternalLink from './external-link';
+import InfoIcon from '../assets/icons/info.svg';
+import WarningIcon from '../assets/icons/warning.svg';
+
+export default {
+  components: {
+    AppTooltip,
+    ExternalLink,
+    InfoIcon,
+    WarningIcon,
+  },
+  props: {
+    image: {
+      type: Object,
+      default() {
+        return {};
+      },
+    },
+    required: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    term: {
+      type: [ Array, String ],
+      required: true,
+    },
+    tooltip: {
+      type: Object,
+      default() {
+        return {};
+      },
+    },
+    type: {
+      type: String,
+      required: false,
+      default: 'string',
+      validator: type => [ 'string', 'link', 'urls', 'image', 'color' ].indexOf(type) !== -1,
+    },
+    validation: {
+      type: Object,
+      default() {
+        return {};
+      },
+    },
+    value: {
+      type: [ Array, Number, String ],
+      required: false,
+      default() {
+        return '';
+      },
+    },
+  },
+  computed: {
+    errorMessage() {
+      return this.errors?.[0]?.message.trim() ?? null;
```

Comment:
I think the `?? null` is obsolete when using optional chaining, because as soon as any part doesn't exist it returns `undefined` which works just as well as `null` in our case.